Misc/NEWS.d/next/Core and Builtins/2019-06-01-23-55-50.bpo-32865.Ha11I1.rst

```diff
@@ -0,0 +1,2 @@
+On Windows, do not use inheritable file descriptors to wrap handles in
+:func:`os.pipe`.
```

Comment:
Would you mind to also document _io._WindowsConsoleIO.fileno() fix?

Reply:
Thank you, Victor, for looking at this PR -- I will update it soon.